### PR TITLE
Create bypass

### PR DESCRIPTION
**Pull request by niranjanakoni**

### CODE REVIEW

**Review by sirmv — CHANGES_REQUESTED** [review]
add some more

**Review by niranjanakoni — COMMENTED** [review]
added more conents

File: bypass [hunk 1]
```diff
@@ -1 +1,3 @@
 asdfhjk
+zxcvb
+zxcvbn
```
Comment (sirmv) [inline bypass:3]:
```suggestion
2nd line
3rd line
```

Comment (sirmv) [inline bypass:3]:
```suggestion
2nd line
3rd line
```

File: bypass [hunk 2]
```diff
@@ -1,3 +1,3 @@
 asdfhjk
-zxcvb
-zxcvbn
+2nd line
+3rd line
```
Comment (sirmv) [inline bypass:3]:
```suggestion
2nd line
3rd line
4th line
```

**Review by sirmv — CHANGES_REQUESTED** [review]

**Review by sirmv — APPROVED** [review]

**Pull request merged**